graylog2-web-interface/webpack/vendor-module-ids.json

```diff
@@ -921,7 +921,30 @@
       "node_modules/simple-swizzle/node_modules/is-arrayish/index.js": 80,
       "node_modules/color/node_modules/color-convert/index.js": 84,
       "node_modules/color/node_modules/color-convert/route.js": 89,
-      "node_modules/styled-theming/index.js": 90
+      "node_modules/styled-theming/index.js": 90,
+      "node_modules/@babel/runtime/helpers/interopRequireDefault.js": 15,
+      "node_modules/react-is/index.js": 18,
+      "node_modules/define-properties/index.js": 30,
+      "node_modules/color-convert/conversions.js": 32,
+      "node_modules/object-keys/index.js": 43,
+      "node_modules/object-keys/isArguments.js": 45,
+      "node_modules/regexp.prototype.flags/implementation.js": 46,
+      "node_modules/regexp.prototype.flags/polyfill.js": 57,
+      "node_modules/color-convert/index.js": 72,
+      "node_modules/color-convert/node_modules/color-name/index.js": 74,
+      "node_modules/color-convert/route.js": 84,
+      "node_modules/object-keys/implementation.js": 89,
+      "node_modules/is-arguments/index.js": 91,
+      "node_modules/object-is/index.js": 92,
+      "node_modules/is-regex/index.js": 93,
+      "node_modules/has/src/index.js": 94,
+      "node_modules/function-bind/index.js": 95,
+      "node_modules/function-bind/implementation.js": 96,
+      "node_modules/regexp.prototype.flags/index.js": 97,
+      "node_modules/regexp.prototype.flags/shim.js": 98,
+      "node_modules/is-date-object/index.js": 99,
+      "node_modules/react-is/cjs/react-is.production.min.js": 100,
+      "node_modules/@babel/runtime/helpers/extends.js": 101
```

Comment:
Why did this change become necessary? Usually the `vendor-module-ids.json` file should only change if either the `vendor.modules.js` or `package.json`/`yarn.lock` changes.

Reply:
It might be related to doing a `yarn install` in a different branch before and not cleaning up before a `yarn build`. In any case, the files should not go into this PR, as it is unrelated to the change.